### PR TITLE
update networking-api-version and update crds

### PR DESCRIPTION
**Pull request by ryansteakley**
**Which issue is resolved by this Pull Request:**

**Description of your changes:**

- Update for kubernetes verison 1.22 and to support eks 1.22  updates apiVersion and follows the new spec

**Testing:**
- [ ] Unit tests pass
- [x] e2e tests pass
- Details about new tests (If this PR adds a new feature)
- Details about any manual tests performed

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: awsconfigs/common/istio-ingress/base/ingress.yaml [hunk 1]
```diff
@@ -12,6 +12,9 @@ spec:
...
       paths:
       - backend:
-          serviceName: istio-ingressgateway
-          servicePort: 80
+          service:
+            name: istio-ingressgateway
+            port:
+              number: 80
         path: /*
+        pathType: ImplementationSpecific
```
Comment (mbaijal) [inline awsconfigs/common/istio-ingress/base/ingress.yaml:20]: How did you pick the value for this field ?

Comment (ryansteakley) [reply awsconfigs/common/istio-ingress/base/ingress.yaml:20]: From kubernetes documentation, `pathType` is now required for each specified path. Options are `Prefix`, `Exact`, and `ImplementationSpecific`. To match the undefined `v1beta1` behavior, use `ImplementationSpecific`. I went with the choice to match the behavior from v1beta1 which we were using previously.

**Comment by surajkota** [comment]
@ryansteakley can we resolve comments and merge https://github.com/awslabs/kubeflow-manifests/pull/319 before this PR?

File: awsconfigs/common/istio-ingress/base/ingress.yaml (hunk 1 above)
Comment (surajkota) [reply awsconfigs/common/istio-ingress/base/ingress.yaml:20]: doc link? just to give read it

Comment (ryansteakley) [reply awsconfigs/common/istio-ingress/base/ingress.yaml:20]: https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingress-v122

**Comment by ryansteakley** [comment]
> @ryansteakley can we resolve comments and merge #319 before this PR?

yes will do that

**Review by mbaijal — APPROVED** [review]

**Review by surajkota — APPROVED** [review]

**Pull request merged**